GamesServices/GamesProvider.Services/GamesPricesService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using AutoMapper;
+using DBAccess;
+using GamesProvider.Services.DTOs;
+using Microsoft.EntityFrameworkCore;
+using Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+
+namespace GamesProvider.Services
+{
+    public class GamesPricesService : IGamesPricesService
+    {
+        private readonly GameServiceDBContext _dbContext;
+        private readonly IMapper _mapper;
+
+        public GamesPricesService(GameServiceDBContext dbContext, IMapper mapper)
+        {
+            _dbContext = dbContext;
+            _mapper = mapper;
+        }
+
+        public GamePriceDTO GetGamePriceById(int gamePriceId)
+        {
+            GamePrices gamePrices = _dbContext.GamePrices
+                .Where(gp => gp.GamePriceId == gamePriceId)
+                .Include(gp => gp.Game)
+                    .ThenInclude(g => g.Images)
+                .Include(g => g.Platform).FirstOrDefault();
+            return _mapper.Map<GamePrices, GamePriceDTO>(gamePrices);
+        }
+        public GamePriceDTO GetGamePriceByPlatformAndGame(int platformId, int gameId)
+        {
+            GamePrices gamePrices = _dbContext.GamePrices
```

Comment:
var

Reply:
Why? You can use var here, but this change is optional

Reply:
By the way, var can be used where you absolutely sure about variable type( for example, like here or if you will have variable with name "country" (type Country, substituted by var) that will call nethod with name GetCountryBySmth()). In all other situations you should mamually specify variable type.

Reply:
By the way, var can be used where you absolutely sure about variable type( for example, like here or if you will have variable with name "country" (type Country, substituted by var) that will call nethod with name GetCountryBySmth()). In all other situations you should mamually specify variable type.